api/members/model.ts

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import {
+    Column,
+    CreateDateColumn,
+    Entity, EntityManager, FindManyOptions,
+    ManyToMany, JoinTable,
+    PrimaryGeneratedColumn,
+    TransactionManager,
+    UpdateDateColumn
+} from "typeorm";
+import EventModel from "../event/model";
+
+
+@Entity({name: 'members'})
+export default class MembersModel {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({length: 64})
+		firstName: string;
+		
+		@Column({length: 64})
+		lastName: string;
+
+		@Column({length: 140})
+		imageUrl: string;
+
+		@Column({length: 140})
+		shortBio: string;
```

Comment:
Again, why 140 when the database column is text?

Reply:
is it okay to leave blank?
@Column({})

Reply:
Yes, you can just to `@column()`. No need to pass in an empty object.